### PR TITLE
Fix the classic double "the the" typos in the ROOT documentation

### PR DESCRIPTION
**Pull request by guitargeek**
Also fixes the related ` a a ` typos.

### CODE REVIEW

**Comment by Axel-Naumann** [comment]
Great! Could we please leave code alone that's not "ours"? E.g. interpreter/llvm, zstd, etc?

**Review by bellenot — APPROVED** [review]
GUI part looks good to me

**Review by lmoneta — APPROVED** [review]
LGTM !
Thank you Jonas for this fix!

**Review by etejedor — APPROVED** [review]

File: math/mathcore/src/mixmax.icc [hunk 1]
```diff
@@ -372,7 +372,7 @@ void seed_uniquestream( rng_state_t* Xin, myID_t clusterID, myID_t machineID, my
     static __thread myuint sumtmp=0;
     static __thread myID_t ID1cached=0, ID2cached=0, ID3cached=0, ID4cached=0;
     
-    if ( ID1cached <=  clusterID &&  ID2cached <= machineID && ID3cached <= runID && ID4cached <= streamID){ // proceed only if the the cached seed value is less than the new seed value
+    if ( ID1cached <=  clusterID &&  ID2cached <= machineID && ID3cached <= runID && ID4cached <= streamID){ // proceed only if the cached seed value is less than the new seed value
```
Comment (Axel-Naumann) [inline math/mathcore/src/mixmax.icc:375]: @lmoneta is it okay if we diverge from "upstream"?

**Review by Axel-Naumann — APPROVED** [review]
LGTM thanks for doing this! I'd like to have @lmoneta confirm regarding mixmax (or exclude it here, @guitargeek )

**Review by couet — APPROVED** [review]
LGTM

**Comment by guitargeek** [comment]
Since Lorenzo is off, I will exclude the change from `math/mathcore/src/mixmax.icc` then. I didn't know it was external code.

I will merge the PR then.

**Pull request merged**